.changeset/rude-camels-attack.md

```diff
@@ -0,0 +1,7 @@
+---
+'@penumbra-zone/query': major
+'@penumbra-zone/types': major
+'@penumbra-zone/wasm': minor
```

Comment:
question: wasm is a major bump too given there's a breaking function signature change right?